includes/Modules/Analytics.php

```diff
@@ -854,23 +854,41 @@ protected function create_data_request( Data_Request $data ) {
 					return true;
 				};
 			case 'GET:report':
-				$date_range = $data['dateRange'] ?: 'last-28-days';
```

Comment:
Let's add a check here to ensure `$data['metrics']` is not empty, and if it is return an error.

Reply:
Updated.